lib/rgeo/impl_helper/basic_geometry_collection_methods.rb

```diff
@@ -177,6 +177,19 @@ def boundary
       def coordinates
         @elements.map(&:coordinates)
       end
+
+      def contains?(rhs)
+        unless Feature::Point === rhs
+          raise(Error::UnsupportedOperation,
+            "Method MultiPolygon#contains? is only defined for Point"
+          )
+        end
+
+        @elements.each do |poly|
+          return true if poly.contains?(rhs)
+        end
+        false
```

Comment:
```suggestion
        @elements.any? { |polygon| polygon.contains?(rhs) }
```

Some rubyification/simplification :)